scripts/examples/python/shell.py

```diff
@@ -103,10 +103,10 @@ def munge_types (str):
     # We want to support directly executing the cmd line,
     # where appropriate
     if len(sys.argv) > cmdAt:
-        cmd = sys.argv[cmdAt]
```

Comment:
There was some warning about it, either maybe feom pylint (short name) the defaults would reject it.

But it might habe also been a name collision (overload) or a method with the same name. 

I would not have been there it it wasn't a because of a warning.